compliance/collection/metrics/metrics.go

```diff
@@ -51,6 +61,20 @@ var (
 			"error",
 		})
 
+	indexDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.ComplianceSubsystem.String(),
+		Name:      "index_duration_seconds",
+		Help:      "Generation duration for Node IndexReports (per Node) in seconds",
+		Buckets:   []float64{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 50, 100, 500, 1000},
+	},
+		[]string{
+			// The Node this scan belongs to
+			"node_name",
+			// Whether the inventory run was completed successfully
+			"error",
+		})
```

Comment:
I know that the other metric uses the same buckets, but I feel that the duration would be typically short, then these numbers may be too high. Can we see some runs that finish under a second?
If yes, then maybe we should add `0.1` and `0.5` to the beginning? (ideally for both v2 and v4)